waveform_editor/gui/editor.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, config):
             active_icon="check",
             description="Save waveform",
             on_click=self.save_waveform,
+            visible=self.param.waveform.rx.pipe(bool),
```

Comment:
Would be even nicer if we could make it depend on self.has_changed (but that one is not declarative) :)